packages/product-json-to-csv/test/__snapshots__/map-product-data.spec.js.snap

```diff
@@ -45,7 +46,10 @@ Array [
   },
   Object {
     "article": "sample 089 WHT",
-    "color": "white",
+    "color.key": 0,
```

Comment:
Why did this change? I would preserve the behavior from the [previous module](https://github.com/sphereio/sphere-node-product-csv-sync/blob/master/src/coffee/exportmapping.coffee) which was mapping lenums like this:
```
attributeName: item key
attributeName.en: English label
``` 
WDYT @Babanila @daern91 ?

Reply:
agreed.
This shouldn't change

![image](https://user-images.githubusercontent.com/462507/43824894-8c9cd95c-9af3-11e8-9f20-226c2679a472.png)

in `map-product-data.spec.js`

Reply:
good catch!